doc/man3/X509_STORE_add_cert.pod

```diff
@@ -55,7 +55,9 @@ operate on pointers to B<X509> objects, though.
 
 X509_STORE_add_cert() and X509_STORE_add_crl() add the respective object
 to the B<X509_STORE>'s local storage.  Untrusted objects should not be
-added in this way.
+added in this way.  The added object's reference count is increased by those
+functions so that the caller need to handle the memory management as well,
+for instance, free the object if needed.
```

Comment:
A little variation, just a suggestion:
```
added in this way.  The added object's reference count is incremented by one,
hence the caller retains ownership of the object and needs to call
X509_STORE_free() when it is no longer needed.
```

Reply:
If you prefer to keep your variant, then I have two little corrections for you:

- 'the caller need' -> 'the caller needs'
- 'free the object if needed.' -> 'free the object if not needed anymore.'

Reply:
In your version, I think `X509_STORE_free()` should be `X509_free()` or other free functions for objects instead of STORE

Reply:
Sorry, I didn't look close enough. Probably it's just better to write `and needs to free it when it is no longer needed.`

Reply:
Sorry for the late response @mspncp , I changed the wording to what your suggested.